core/src/main/resources/hudson/model/Cause/UserIdCause/description.properties

```diff
@@ -20,5 +20,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-started_by_user=Started by user <a href="{2}/user/{0}">{1}</a>
+started_by_user=Started by user <a href="{2}/{0}">{1}</a>
```

Comment:
Does it generate the correct number of slashes in URL?

Reply:
It does not include a leading slash, so, yes.